### PR TITLE
docs: Adding Concourse resource for trivy

### PR DESCRIPTION
**Pull request by hriprsd**
## Related issues
- Close #XXX

## Related PRs
- [ ] #XXX
- [ ] #YYY

Remove this section if you don't have related PRs.

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [ ] I've added tests that prove my fix is effective or that my feature works.
- [x] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: docs/ecosystem/cicd.md [hunk 1]
```diff
@@ -57,3 +57,11 @@ pipeline:
 ```
 
 Woodpecker does use Trivy itself so you can [see it in use there](https://github.com/woodpecker-ci/woodpecker/pull/1163).
+
+## Concourse CI (Community)
+[Concourse CI](https://concourse-ci.org/) is a CI/CD service.
+
+You can use Trivy Resource in Concourse for scanning containers and lets you  introduce security scanning into your workflow.
```
Comment (itaysk) [inline docs/ecosystem/cicd.md:64]:
```suggestion
You can use Trivy Resource in Concourse for scanning containers and introducing security scanning into your workflow.
```

File: docs/ecosystem/cicd.md [hunk 2]
```diff
@@ -57,3 +57,11 @@ pipeline:
...
+It has capability to fail the pipeline, create issues, alert communication channels (using respective resources) based on trivy scan output.
```
Comment (itaysk) [inline docs/ecosystem/cicd.md:65]:
```suggestion
It has capabilities to fail the pipeline, create issues, alert communication channels (using respective resources) based on Trivy scan output.
```

**Review by AnaisUrlichs — APPROVED** [review]

**Comment by knqyf263** [comment]
I'll merge it after @itaysk approves that.

**Comment by knqyf263** [comment]
@hriprsd Thanks!

**Pull request merged**